components/modal_controller/modal_controller.tsx

```diff
@@ -43,16 +45,41 @@ export default class ModalController extends React.PureComponent<Props> {
 
         const modalOutput = [];
 
+        const activeElement = document.activeElement;
```

Comment:
This needs to be done somewhere else since this will change any time the `ModalController` happens to re-render. That's not too much of a problem at the moment, but I could see this getting broken like when this component is migrated to a functional component at some point. When that happens, this logic will break because `activeElement` could change after the modal is opened.

Instead, I think we could move `activeElement` so that its value is set in `openModal` and then its saved into Redux so that its value never changes after that. We can just save it per-modal in the same part of the store as `dialogType` and `dialogProps`